apps/src/music/views/BeatPad.jsx

```diff
@@ -1,72 +1,74 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import {useSelector} from 'react-redux';
 import classNames from 'classnames';
 import styles from './beatpad.module.scss';
-import FontAwesome from '@cdo/apps/templates/FontAwesome';
-import musicI18n from '../locale';
+import {BlockTypes} from '../blockly/blockTypes';
+import {TRIGGER_FIELD} from '../blockly/constants';
 
-const BUTTONS_PER_ROW = 3;
+const BUTTONS_PER_ROW = 4;
 const enabledClasses = [
-  styles.purple,
-  styles.strawberry,
-  styles.orange,
-  styles.yellow,
-  styles.green,
-  styles.teal,
+  styles.greenTrigger,
+  styles.greenTrigger,
+  styles.greenTrigger,
+  styles.greenTrigger,
+  styles.greenTrigger,
+  styles.greenTrigger,
 ];
 
 /**
  * Renders the Beat Pad component, which can be used to play numbered triggers during playback
  */
-const BeatPad = ({triggers, playTrigger, onClose, isPlaying}) => {
-  const renderTriggers = () => {
-    const rows = [];
+const BeatPad = ({triggers, playTrigger, onClose, isPlaying, hasTrigger}) => {
+  const selectedBlockId = useSelector(state => state.music.selectedBlockId);
+  const selectedBlock =
+    selectedBlockId && Blockly.mainBlockSpace.getBlockById(selectedBlockId);
+  const isSelectedBlockTriggerAt =
+    selectedBlock?.type === BlockTypes.TRIGGERED_AT_SIMPLE2;
+  const selectedTriggerId =
+    isSelectedBlockTriggerAt && selectedBlock?.getFieldValue(TRIGGER_FIELD);
```

Comment:
Optional - could move this into music/blockly/blockUtils as a `getSelectedTriggerIdForBlockId(blockId)` or as its own selector inside musicRedux. Also fine keeping it here, just thinking if would need this calculation elsewhere.